src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderMap.cs

```diff
@@ -47,59 +75,152 @@ private CoseHeaderMap(bool isReadOnly)
             }
         }
 
+        /// <summary>
+        /// Gets the CBOR encoded value associated with the specified label, as a signed integer.
+        /// </summary>
+        /// <param name="label">The label of the value to get.</param>
+        /// <returns>The decoded value associated with the specified label, as a signed integer.</returns>
+        /// <exception cref="InvalidOperationException">Decodification failed, see the inner exception for the exact exception that was thrown.
```

Comment:
I don't think we've ever used the word "Decodification" in docs, and I don't think we ever (or at least "usually") say to see the inner exception.

```suggestion
        /// <exception cref="InvalidOperationException"><paramref name="label" /> was present in the header map, but the value could not be decoded as a 32-bit signed integer.</exception>
```

(trailing data would be a variant reason why it could not be decoded, so I don't think it's worth mentioning)

Reply:
Why do you want to mention "`<paramref name="label" /> was present in the header map`"?
Actually, this method also needs to specify a KeyNotFoundException.

Reply:
Fair, "The value associated with the specified label could not be decoded as a 32-bit signed integer" seems reasonable.